corehq/apps/reports/tasks.py

```diff
@@ -145,40 +142,6 @@ def monthly_reports():
         send_delayed_report(report_id)
 
 
-@periodic_task(run_every=crontab(hour="23", minute="59", day_of_week="*"), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
-def saved_exports():
-    for group_config_id in get_doc_ids_by_class(HQGroupExportConfiguration):
-        export_for_group_async.delay(group_config_id)
-
-    for daily_saved_export_id in get_all_daily_saved_export_instance_ids():
-        from corehq.apps.export.tasks import rebuild_export_task
-        last_access_cutoff = datetime.utcnow() - timedelta(days=settings.SAVED_EXPORT_ACCESS_CUTOFF)
-        rebuild_export_task.apply_async(
-            args=[
-                daily_saved_export_id, last_access_cutoff
-            ],
-            # Normally the rebuild_export_task uses the background queue,
-            # however we want to override it to use its own queue so that it does
-            # not disrupt other actions.
-            queue=SAVED_EXPORTS_QUEUE,
-        )
-
-
-@task(queue='background_queue', ignore_result=True)
-def rebuild_export_task(groupexport_id, index, last_access_cutoff=None, filter=None):
```

Comment:
Note that this is `corehq.apps.reports.tasks.rebuild_export_task`, which is _not_ the `corehq.apps.export.tasks.rebuild_export_task` called above.  It doesn't appear to be called anywhere, so this commit drops it.